org.eclim.groovy/java/org/eclim/plugin/groovy/command/complete/CodeCompleteCommand.java

```diff
@@ -61,7 +69,8 @@
     "REQUIRED f file ARG," +
     "REQUIRED o offset ARG," +
     "REQUIRED e encoding ARG," +
-    "REQUIRED l layout ARG"
+    "REQUIRED l layout ARG," +
+    "OPTIONAL j javaDoc ARG"
```

Comment:
Can you update this to be a `NOARG` and just use the presence of the arg to indicate `true`.